src/ProjectFileTools/GoToDefinition/GoToDefinitionController.cs

```diff
@@ -41,11 +44,55 @@ public int QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, Int
             return Next?.QueryStatus(pguidCmdGroup, cCmds, prgCmds, pCmdText) ?? (int)Microsoft.VisualStudio.OLE.Interop.Constants.OLECMDERR_E_NOTSUPPORTED;
         }
 
+        private static bool PackageExistsOnNuGet(string packageName, string version, out string url)
+        {
+            string packageAndVersionUrl = $"https://www.nuget.org/packages/{packageName}/{version}/";
```

Comment:
It would be nice if the logic of finding the Nuget package would reside inside the .Nuget project.

On a side note, the Definition class is intended to be an abstraction level for the set of matches. However, the implementation currently is very limited (we would need to implement what we were calling Resolvers; that would allow us to write this code in a more generic way).

For now, I think that is fine.

Reply:
Filed #29 to track moving the logic